test/modules/GenericMqttTester/src/main.rs

```diff
@@ -12,10 +20,44 @@ async fn main() -> Result<()> {
 
     let settings = Settings::new()?;
 
-    todo!()
+    let tester = MessageTester::new(settings).await?;
+    let tester_shutdown = tester.shutdown_handle();
+
+    // TODO: We should be able to change this pattern to spawn only shutdown future (which blocks), but this doesn't work, Need to figure out why.
```

Comment:
There is a significant problem with the below two futures. Ideally we should spawn the blocking shutdown future and not spawn the test future. This doesn't work and the client does not get polled.